bench/tests/test_base.py

```diff
@@ -13,7 +13,7 @@
 import bench.utils
 
 if sys.version_info.major == 2:
-	FRAPPE_BRANCH = "version-12"
+	FRAPPE_BRANCH = "version-13"
```

Comment:
Why setup v13 Frappe for PY2? They're incompatible.

Reply:
Right, will make changes

Reply:
Changed develop to v13 instead